torntools/scripts/content/stockexchange/ttStockexchange.js

```diff
@@ -504,3 +507,86 @@ function showTotalPortfolioValue() {
 		);
 	}
 }
+
+function hideStockBlocks() {
+	if (hide_stock_blocks.length) {
+		for (let hideStockBlock of hide_stock_blocks) {
+			doc.find(`div.stock-main-wrap li.item.item-wrap[data-stock=${hideStockBlock.toLowerCase()}]`).style.display = "none";
+		}
+		let rawHTML =
+			'<div class="info-msg-cont gray border-round m-top10"><div class="info-msg border-round" style="background-color: #627e0d;"><i class="info-icon"></i><div class="delimiter"><div class="msg right-round" style="background-color: #627e0d;color: #627e0d;">Some stock blocks have been disabled by TornTools. Please re-enable them in Settings.</div></div></div></div>';
+		doc.find("div.stock-main-wrap div.title-black.top-round.m-top10").insertAdjacentHTML("beforeBegin", rawHTML);
+	}
+}
+
+function hidePortfolio() {
+	let buttonHTML = "<button class='hide-portfolio'>Hide this stock block</button>";
```

Comment:
I would shorten it to `Hide`. It's on the portfolio, on each individual stock block, people should know on their own imo.

Reply:
I can also imagine it not going well on mobile if it's this long

Reply:
> I would shorten it to `Hide`. It's on the portfolio, on each individual stock block, people should know on their own imo.

Well, meaningful names may help future developers recognise the usage of a code. We don't document our code so, that might help a bit.

Reply:
It's about the inner text of the button though: ![Screenshot](https://i.imgur.com/x2ZiZBR.png)

Reply:
Could also happen on PC, but very unlikely (100s of billions in stock)

Reply:
Changed.